lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -656,6 +663,21 @@ def set_severity(record, entry)
       end
     end
 
+    def set_http_request(record, entry)
+      return unless record.key?('httpRequest')
```

Comment:
you should also return if !record['httpRequest'].is_a?(Hash) or it could throw an exception.  Please add a test for this as well.